admin_manual/installation/nginx.rst

```diff
@@ -1,10 +1,14 @@
 ===================
-Nginx configuration
+NGINX configuration
 ===================
 
-This page covers example Nginx configurations to use with running a Nextcloud
-server. These configurations examples were originally provided by
-`@josh4trunks <https://github.com/josh4trunks>`_ and are community-maintained. (Thank you contributors!)
+.. warning::
```

Comment:
I'm just wondering why we have `.. subtitle::`?
I'm not familiar enough with RST format to know what that is trying to achieve?

Reply:
I don't see the subtitle in here. Or are you referring to another part of that document?